app/views/xur.html

```diff
@@ -4,10 +4,10 @@
     <div ng-repeat="category in vm.categoryOrder">
       <h2 ng-bind="category"></h2>
       <div class="items">
-        <div class="xurItem" ng-repeat="saleItem in vm.itemCategories[category] track by saleItem.item.index">
+        <div class="xurItem" align="right" ng-repeat="saleItem in vm.itemCategories[category] track by saleItem.item.index">
           <dim-simple-item id="xur-{{::saleItem.item.index}}" item-data="saleItem.item" ng-click="vm.itemClicked(saleItem.item, $event)"></dim-simple-item>
           <div class="cost" ng-class="{notenough: (vm.totalCoins[saleItem.currency.itemHash] < saleItem.cost)}">
-            {{::saleItem.cost}}/{{vm.totalCoins[saleItem.currency.itemHash]}}
+            {{::saleItem.cost}}
```

Comment:
It would be interesting to show a 'materials needed' section.  Some factions use glimmer/marks/coins/etc.  I could get behind this.